src/controllers/helpers/deductionsOperations.js

```diff
@@ -0,0 +1,66 @@
+const convertToCamelCase = require('camelcase-keys-deep');
+const taxes = require('./taxes');
+
+const contributionsOperations = (bonus, contributionEmployee, contributionInstitute) => {
+  const {
+    salary,
+    childAllownace,
+    wifeAllowance,
+    transportation,
+    allowanceGasoline,
+    allowanceMobile,
+    allowanceJerusalem,
+  } = bonus;
+
+  const contributionOfInstitute = (salary
+    + wifeAllowance
+    + childAllownace
+    + allowanceGasoline
+    + allowanceMobile
+    + transportation
+    + allowanceJerusalem)
+  * contributionInstitute;
```

Comment:
***indentation***
add a single white line after each style block
**same for the rest of the file**